Dockerfile.multistage

```diff
@@ -0,0 +1,46 @@
+FROM golang:1.17-stretch
+
+WORKDIR /go/src/app
+
+COPY go.sum go.mod ./
+RUN go mod download
+
+COPY . .
+RUN make build
+
+FROM debian:stretch-slim AS container-agent
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV GODEBUG http2client=0
+
+RUN apt-get update -yq && \
+    apt-get install -yq --no-install-recommends ca-certificates sudo && \
+    apt-get clean && \
```

Comment:
Maybe `apt-get clean` is not required.

> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run